modulify-idea.md

```diff
@@ -0,0 +1,126 @@
+# Ideas for Modulification
```

Comment:
I think this file was included from the other branch by accident?

Reply:
Resolved by rebasing + force-pushing.